src/Http/Environment.php

```diff
@@ -1008,12 +995,22 @@ public function toArray(): array
             'isBehindProxy' => $this->isBehindProxy,
             'path'          => $this->path,
             'port'          => $this->port,
-            'requestUri'    => $this->requestUri,
+            'requestUri'    => $this->requestUrl(),
             'scriptPath'    => $this->scriptPath,
             'url'           => $this->url
         ];
     }
 
+    /**
+     * Returns the full base URL object
+     *
+     * @return \Kirby\Http\Uri
+     */
+    public function uri()
```

Comment:
I think the name `uri()` is very easy to confuse. It could both be the the base URI or the full request URI. What about `baseUri()`?

Also: For consistency, the `requestUri()` could return a `Http\Uri` object as well. I know that this out of scope of this PR, but we can never make such consistency fixes again after 3.7 without breaking changes.

Reply:
I already refactored the requestUri method to return an URI object as well. 

We now have: 

```php
$environment->url()
$environment->uri()

$environment->requestUrl()
$environment->requestUri()
```

I like the idea to rename uri and url to base uri and base url. Will take care of it.